dev-env/windows/manifests/msys2.json

```diff
@@ -31,6 +33,7 @@
             "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /netcat.msys2'\"",
             "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /patch.msys2'\"",
             "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /openssl.msys2'\"",
+            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /python.msys2'\"",
```

Comment:
We have a separate Python entry in `dev-env/windows/manifests/python-3.6.7.json`. Why add another Python instance here?

Reply:
We do, but the build fails otherwise because PostgreSQL depends on Python.

Reply:
I see, so it seems that `PostgreSQL` is unable to find the scoop provided Python and only finds the msys2 provided Python, correct?

Another option could be to replace the scoop Python by the msys2 Python to avoid this duplication. That may also fix an issue we've been having with Python on Windows: Scoop cannot uninstall Python, the Python package that scoop installs can only be uninstalled manually. So, if a Python installation fails, say due to network issues, then the dev-env's automatic reinstall does not work and we have to kill the affected CI node. If msys2 doesn't have that limitation, then that would be an easy way to fix this issue. @garyverhaegen-da WDYT?

Reply:
All for not having to kill nodes. I'm not sure I understand why the scoop version can't be uninstalled and the msys2 version might, but happy to make the switch and see what happen.

Reply:
Cool. I'll try and switch it in a separate PR and see if it works.

Reply:
> All for not having to kill nodes. I'm not sure I understand why the scoop version can't be uninstalled and the msys2 version might, but happy to make the switch and see what happen.

Looks to be working, I tried out the following on Windows:
* checkout this PR
* Run `dadew sync` (installing python in msys2)
* checkout master
* Run `dadew sync` (uninstall python in msys2)

That worked without trouble.

`msys2` uses `pacman` as a package manager. AFAIK the scoop package uses Python's own Windows installer. The latter has some limitation that makes it not work in unattended mode. Thought, I don't know the deeper reasons for that.

Reply:
Good enough for me.

Reply:
Looked into it and I can't really do much without a Windows machine for testing, unfortunately. I wouldn't even know how to start configuring `pipenv`.